### ISSUE TITLE
1.38.0 fails to compile bootstrap

### ISSUE THREAD

**Issue by folknor** [sass/dart-sass#1441]
Hi

Apologies if this is a known issue, or if this is a planned change, or if I'm wrong about this. I could not find any other reference to this problem in your issue tracker or Bootstraps issue tracker.

Essentially, short and sweet;
1. I upgraded from sass 1.37.5 to 1.38.0 and the build broke
2. I downgraded to 1.37.5 and it works again

Please let me know if there's anything I should test or any information you need.

Hopefully the terminal output below can be helpful.

Thank you!

```
~/test $ pnpm up
Packages: +68 -54
++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++++------------------------------------------------------
Downloading registry.npmjs.org/@parcel/transformer-js/2.0.0-nightly.802: 47.1 MB/47.1 MB, done
Progress: resolved 744, reused 690, downloaded 54, added 68, done
node_modules/.pnpm/core-js@3.16.2/node_modules/core-js: Running postinstall script, done in 29ms

devDependencies:
- sass 1.37.5
+ sass 1.38.0
~/test $ npm run deploy

> test@1.0.0 deploy
> NODE_ENV=production pnpm run build && pnpm run css:purge -s


> test@1.0.0 build /home/test
> pnpm run clean -s && pnpm run copy -s && pnpm run parcel:build


> test@1.0.0 parcel:build /home/test
> parcel build --detailed-report 0 --target web index.html

🚨 Build failed.

@parcel/transformer-sass: Can't find stylesheet to import.
   ╷
10 │ @import "functions";
   │         ^^^^^^^^^^^
   ╵
  file:/home/test/node_modules/.pnpm/bootstrap@5.1.0/node_modules/bootstrap/scss/bootstrap.scss 10:9  @import
  sass/main.scss 10:9                                                                                                                                              root stylesheet

  Error: Can't find stylesheet to import.
  ╷
  10 │ @import "functions";
  │         ^^^^^^^^^^^
  ╵
  file:/home/test/node_modules/.pnpm/bootstrap@5.1.0/node_modules/bootstrap/scss/bootstrap.scss 10:9  @import
  sass/main.scss 10:9                                                                                                                                              root stylesheet
  at Object._newRenderError (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:13226:19)
  at Object._wrapException (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:13046:16)
  at _render_closure1.call$2 (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:82175:21)
  at _RootZone.runBinary$3$3 (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:27558:18)
  at _FutureListener.handleError$1 (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:26107:19)
  at _Future__propagateToListeners_handleError.call$0 (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:26405:49)
  at Object._Future__propagateToListeners (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:4546:77)
  at _Future._completeError$2 (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:26237:9)
  at _AsyncAwaitCompleter.completeError$2 (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:25891:12)
  at Object._asyncRethrow (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:4345:17)
 ERROR  Command failed with exit code 1.
 ERROR  Command failed with exit code 1.
~/test $ pnpm prune                                                                                                                                                                                                                                       ✘ 1 
Lockfile is up-to-date, resolution step is skipped
Already up-to-date

node_modules:
- @babel/plugin-transform-flow-strip-types 7.14.5
- @babel/types 7.15.0
- @parcel/types 2.0.0-nightly.802
- @types/http-proxy 1.17.7
- @types/node 16.6.1
- mime-types 2.1.18
- utility-types 3.10.0
~/test $ pnpm up
Packages: +1 -1
+-
Progress: resolved 744, reused 744, downloaded 0, added 1, done

devDependencies:
- sass 1.38.0
+ sass 1.37.5 (1.38.0 is available)
~/test $ pnpm run deploy

> test@1.0.0 deploy /home/test
> NODE_ENV=production pnpm run build && pnpm run css:purge -s


> test@1.0.0 build /home/test
> pnpm run clean -s && pnpm run copy -s && pnpm run parcel:build


> test@1.0.0 parcel:build /home/test
> parcel build --detailed-report 0 --target web index.html

⠧ Building index.html...
WARNING: Using / for division is deprecated and will be removed in Dart Sass 2.0.0.

Recommendation: math.div(20em, 16)

More info and automated migrator: https://sass-lang.com/d/slash-div

   ╷
12 │ $fa-fw-width:          (20em / 16);
   │                         ^^^^^^^^^
   ╵
    node_modules/@fortawesome/fontawesome-free/scss/_variables.scss 12:25  @import
    sass/main.scss 12:9                                                    root stylesheet

✨ Built in 4.01s

dist/index.html                      36.53 KB    498ms
dist/header.e5eda4e6.webm           620.79 KB    1.05s
dist/header.63cd4631.mp4            710.05 KB     54ms
dist/1.3bdf9607.webm                725.55 KB     29ms
dist/1.52a0c7d1.mp4                 780.96 KB     48ms
dist/2.4e8aaacd.webm                620.55 KB     56ms
dist/2.2ae0db1a.mp4                 610.65 KB    347ms
dist/index.70d05786.css             234.01 KB    1.16s
dist/cerapro-heavy.7e31ef0c.ttf     159.92 KB     54ms
dist/labelle.91b50f3c.ttf            52.23 KB     29ms
dist/test.1a2d2a9b.ttf              284.89 KB     48ms
dist/five.823a1d31.jpg              458.66 KB     55ms
dist/fa-solid-900.6002b354.woff2     76.43 KB    346ms
dist/fa-solid-900.80005ece.woff      99.27 KB    1.05s
dist/one.b41dff52.jpg                265.7 KB     53ms
~/test $ pnpm up
Packages: +1 -1
+-
Progress: resolved 744, reused 744, downloaded 0, added 0, done

devDependencies:
- sass 1.37.5
+ sass 1.38.0
~/test $ pnpm run deploy

> test@1.0.0 deploy /home/test
> NODE_ENV=production pnpm run build && pnpm run css:purge -s


> test@1.0.0 build /home/test
> pnpm run clean -s && pnpm run copy -s && pnpm run parcel:build


> test@1.0.0 parcel:build /home/test
> parcel build --detailed-report 0 --target web index.html

🚨 Build failed.

@parcel/transformer-sass: Can't find stylesheet to import.
   ╷
10 │ @import "functions";
   │         ^^^^^^^^^^^
   ╵
  file:/home/test/node_modules/.pnpm/bootstrap@5.1.0/node_modules/bootstrap/scss/bootstrap.scss 10:9  @import
  sass/main.scss 10:9                                                                                                                                              root stylesheet

  Error: Can't find stylesheet to import.
  ╷
  10 │ @import "functions";
  │         ^^^^^^^^^^^
  ╵
  file:/home/test/node_modules/.pnpm/bootstrap@5.1.0/node_modules/bootstrap/scss/bootstrap.scss 10:9  @import
  sass/main.scss 10:9                                                                                                                                              root stylesheet
  at Object._newRenderError (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:13226:19)
  at Object._wrapException (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:13046:16)
  at _render_closure1.call$2 (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:82175:21)
  at _RootZone.runBinary$3$3 (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:27558:18)
  at _FutureListener.handleError$1 (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:26107:19)
  at _Future__propagateToListeners_handleError.call$0 (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:26405:49)
  at Object._Future__propagateToListeners (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:4546:77)
  at _Future._completeError$2 (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:26237:9)
  at _AsyncAwaitCompleter.completeError$2 (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:25891:12)
  at Object._asyncRethrow (/home/test/node_modules/.pnpm/sass@1.38.0/node_modules/sass/sass.dart.js:4345:17)
 ERROR  Command failed with exit code 1.
 ERROR  Command failed with exit code 1.
 ERROR  Command failed with exit code 1.
~/test $
```

**Comment by hendotcat** [comment]
Oh interesting! I tried the following just now locally.

```
git init ~/dart-sass-issue-1441
cd ~/dart-sass-issue-1441
yarn add sass@1.38.0 bootstrap@5.1.0
yarn sass node_modules/bootstrap/scss/bootstrap.scss
```

It worked fine though. So next I tracked down the CI build that ran when dependabot bumped Sass to v1.38.0 over on the Bootstrap project and it seems to have succeeded fine there too: https://github.com/twbs/bootstrap/runs/3348728782

Looks to me like this is probably an issue with your particular build tools or config rather than a Sass issue! Sorry if that's not a mega helpful answer but hopefully it helps you at least eliminate one possible cause of the problem!
